fuse_tutorials/config/state_estimator.yaml

```diff
@@ -0,0 +1,61 @@
+# Fixed-lag smoother configuration
+optimization_frequency: 10
+transaction_timeout: 0.1
+lag_duration: 1.0
+
+motion_models:
+  unicycle_motion_model:
+    type: fuse_models::Unicycle2D
+
+sensor_models:
+  initial_localization_sensor:
+    type: fuse_models::Unicycle2DIgnition
+    motion_models: [unicycle_motion_model]
+    ignition: true
+  odometry_sensor:
+    type: fuse_models::Odometry2D
+    motion_models: [unicycle_motion_model]
+  imu_sensor:
+    type: fuse_models::Imu2D
+    motion_models: [unicycle_motion_model]
+  range_sensor:
+    type: fuse_tutorials::RangeSensorModel
+    motion_models: [unicycle_motion_model]
+
+publishers:
+  odom_publisher:
+    type: fuse_models::Odometry2DPublisher
+  beacon_publisher:
+    type: fuse_tutorials::BeaconPublisher
+
+unicycle_motion_model:
+  #                          x      y      yaw    vx     vy     vyaw   ax     ay
+  process_noise_diagonal: [0.100, 0.100, 0.100, 0.100, 0.100, 0.100, 0.100, 0.100]
+  buffer_length: 10.0
+
+initial_localization_sensor:
+  publish_on_startup: false
+
+odometry_sensor:
+  topic: wheel_odom
+  linear_velocity_dimensions: ['x', 'y']
+  angular_velocity_dimensions: ['yaw']
+
+imu_sensor:
+  topic: imu
+  angular_velocity_dimensions: ['yaw']
+
+range_sensor:
+  {}
+
+odom_publisher:
+  topic: odom
+  base_link_frame_id: base_link
+  odom_frame_id: odom
+  map_frame_id: map
+  world_frame_id: map
+  publish_tf: false
+
+beacon_publisher:
+  map_frame_id: map
+  
```

Comment:
![image](https://user-images.githubusercontent.com/5042191/136060235-2e5d9bdc-6f29-4c95-942d-233829e8565c.png)

GH says there is no newline at the end of this file...after the new line at the end of this file.